canonical/ubuntu-advantage-client
# Release 27.3

Created with `git rebase --onto upstream/release-27 664c0795d8ca6fe81301b17d809106e8ad4bd64b` and fixing changelog conflicts (where that commit hash is the last commit in the main branch that release-27 already had in order)

## TODO
- [x] update with new commits as last few PRs land
    - [x] include LP references
- [x] fix line 154 in debian/changelog to match what is in main and in pkg/ubuntu/devel

## To Test/Review:
`git diff main` to make sure the contents match what we have in main

`git diff pkg/ubuntu/devel -- debian/changelog` to check the changelog diff

Review changelog using the checklist in release docs:
- [ ] Is the version correctly updated ? We must ensure that the new version on the changelog is correct and it also targets the latest Ubuntu release at the moment.
- [ ] Is the entry useful for the user ? The changelog entries should be user focused, meaning that we should only add entries that we think users will care about (i.e. we don't need entries when fixing a test, as this doesn't provide meaningful information to the user)
- [ ] Is this entry redundant ? Sometimes we may have changes that affect separate modules of the code. We should have an entry only for the module that was most affected by it
- [ ] Is the changelog entry unique ? We need to verify that the changelog entry is not already reflected in an earlier version of the changelog. If it is, we need not only to remove but double check the process we are using to cherry-pick the commits
- [ ] Is this entry actually reflected on the code ? Sometimes, we can have changelog entries that are not reflected in the code anymore. This can happen during development when we are still unsure about the behavior of a feature or when we fix a bug that removes the code that was added. We must verify each changelog entry that is added to be sure of their presence in the product.
